VimFlavor

```diff
@@ -0,0 +1 @@
+flavor 'fatih/vim-go', '~> 1.5'
```

Comment:
This line should be removed since it means `This test depends on external plugin vim-go`